Documentation/ceph-upgrade.md

```diff
@@ -62,6 +62,12 @@ git clone --single-branch --depth=1 --branch v1.7.4 https://github.com/rook/rook
 cd rook/cluster/examples/kubernetes/ceph
 ```
 
+**IMPORTANT** If you have CephFS volumes and are upgrading from Rook v1.6.0 - v1.6.4,
```

Comment:
```suggestion
**IMPORTANT** If you have CephFS or RBD volumes and are upgrading from Rook v1.6.0 - v1.6.4,
```

Write hangs for RBD volumes, see https://github.com/rook/rook/issues/8085#issuecomment-858578417